pkg/controllers/vscr/podpolling_reconciler.go

```diff
@@ -73,16 +73,21 @@ func (p *PodPollingReconciler) checkScrutinizeContainerStatus(ctx context.Contex
 	if cntStatus == nil {
 		return ctrl.Result{}, fmt.Errorf("could not find scrutinize container status")
 	}
+	stat := &v1beta1.VerticaScrutinizeStatus{}
+	stat.PodName = p.Vscr.Status.PodName
+	stat.PodUID = p.Vscr.Status.PodUID
```

Comment:
Can we just do a copy here? Then we don't need to enumerate each field.
```suggestion
	stat := p.Vscr.Status.DeepCopy()
```